### ISSUE TITLE
SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by rajeshisnepali** [TheWeirdDev/Bluetooth_Headset_Battery_Level#10]
```
File "./bl_battery.py", line 42
    print(f"Battery level for {device} is {blevel}%")
                                                   ^
SyntaxError: invalid syntax

```

My syntax is `./bl_battery.py 00:00:00:00:00:00`

**Comment by TheWeirdDev** [comment]
You need to upgrade python to version 3.6.0 or newer

**Comment by rajeshisnepali** [comment]
> You need to upgrade python to version 3.6.0 or newer

I thought it'd work to version >3. Mine is Python 3.5.2. I will update you when completed.

**Comment by TheWeirdDev** [comment]
Yes i forgot to mention it in the description. I added it now.

**Comment by rajeshisnepali** [comment]
Is there anything missing?

![2020-04-13_18-27](https://user-images.githubusercontent.com/23610013/79121233-8bb98d00-7db4-11ea-8a41-a084fa67f72a.png)

**Comment by TheWeirdDev** [comment]
Do you have more than one version of python3 installed?
You can run the script directly with the newer version:
```
python3.7 ./bl_battery [MAC]
```

or you can change the first line of the script to match the version. for example:

```
#!/usr/bin/env python3.7
```

**Comment by rajeshisnepali** [comment]
```
Traceback (most recent call last):
  File "./bl_battery.py", line 68, in <module>
    main()
  File "./bl_battery.py", line 57, in main
    s = socket.socket(socket.AF_BLUETOOTH,
AttributeError: module 'socket' has no attribute 'AF_BLUETOOTH'

```

I ran `python3.7 ./bl_battery [MAC]`

**Comment by TheWeirdDev** [comment]
This is the same issue as #9.
Your python binary was built without Bluetooth support. You need to install it from another repository or compile the python source code from the official website.

**Comment by rajeshisnepali** [comment]
I ran to an unexpected error. I reinstalled 3.7.4 from the official source but I think I messed up with pip3. 
 It says missing `dbus-python` and neither allows me to install `dbus-python` nor configure it.

But that's okay. That's not the problem with this code, I know.  I'll come to install it later. 

I hope any problem doesn't arise when installed all modules. 
Anyways, Thanks for the code.
